packages/flame/lib/src/components/spawn_component.dart

```diff
@@ -51,8 +51,8 @@ class SpawnComponent extends Component {
   /// spawns and [maxPeriod] will be the maximum amount of time before it
   /// spawns.
   SpawnComponent.periodRange({
-    required double minPeriod,
-    required double maxPeriod,
+    required double this.minPeriod,
+    required double this.maxPeriod,
```

Comment:
is it possible to detect this bug in a test case?

Reply:
Added a test for it!